images/build/Dockerfile

```diff
@@ -11,6 +11,7 @@ ENV LANG C.UTF-8
 
 # Install basic build tools
 RUN apt-get update \
+    && apt-get upgrade -y \
```

Comment:
We are doing an 'upgrade' to get latest of the installed packages. These installed packages could be from the base image, but if the base image is not update yet with any latest updates to the packages it contains, we might run into risk of being flagged by S360 team.